xml/System.Windows.Controls/TabControl.xml

```diff
@@ -289,9 +289,13 @@
  The following example uses the <xref:System.Windows.Controls.TabControl.ContentTemplateSelector%2A> property to display the content of a <xref:System.Windows.Controls.TabItem> differently, depending on the hometown of the selected person.  
   
  [!code-xaml[TabControlContentTemplateSelector#1](~/samples/snippets/csharp/VS_Snippets_Wpf/TabControlContentTemplateSelector/CSharp/Window1.xaml#1)]  
```

Comment:
This separation of two XAML fragments seems confusing. I'd consider combining them for both C# and VB.